public/app/themes/justice/src/js/login.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Modify the login page
+ */
+
+const wp_link = document.querySelector('a[href="https://en-gb.wordpress.org/"]');
+const site_link = document.querySelector('#backtoblog a');
+
+if (wp_link && site_link) {
+    wp_link.href = site_link.href
+}
```

Comment:
Is there a way to see this working? I can't see it on the standard login page.

Reply:
I hate camelCase lol

Reply:
It changes the href of the logo link, to make sure we are not taking stakeholders to WordPress - if they click it they go to the home page of the site.